src/Modules/Admin/Commands.Admin/help/Get-PowerBIArtifact.md

```diff
@@ -0,0 +1,101 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Get-PowerBIArtifact
+
+## SYNOPSIS
+Retrieves artifacts belonging to Power BI workspaces.
+
+## SYNTAX
+
+### List
+```
+Get-PowerBIArtifact [-First <Int32>] [-Skip <Int32>] [<CommonParameters>]
+```
+
+### All
+```
+Get-PowerBIArtifact [-All] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Retrieves a list of Power BI workspaces with information about reports, dashboards, datasets, dataflows and workbooks.
+By default (without -First parameter) it shows the first 100 workspaces. Use -First and -Skip to fetch more workspaces or use -All to return all workspaces.
+
+Before you run this command, make sure you log in using Connect-PowerBIServiceAccount.
+This cmdlet requires the calling user to be a tenant administrator of the Power BI service.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```

Comment:
If we keep this cmdlet, this would need to be filled out

Reply:
No more applicable